src/zone.rs

```diff
@@ -35,6 +35,33 @@ impl<'a> Endpoint<Zone> for ZoneDetails<'a> {
     }
 }
 
+
+/// Zone Details
+/// https://api.cloudflare.com/#zone-purge-files-by-cache-tags-or-host
+pub struct ZoneCachePurge<'a> {
```

Comment:
To be consistent with documentation and the rest of the codebase, it'd probably be better for this endpoint to be called `ZonePurgeFiles`.